src/main/kotlin/com/bookmarks/models/User.kt

```diff
@@ -0,0 +1,14 @@
+package com.bookmarks.models
+
+data class User(
+    val id: UInt,
+    val name: String,
+    val surname: String,
+    val nickname: String,
+    val subscriptions: MutableSet<Subscription>,
+    val purchasedBookIds: MutableSet<UInt>,
```

Comment:
It is a bad tone to have `Mutable*` something in data class. It may lead to a really unexpected behavior.

Reply:
Fixed.